src/charm.py

```diff
@@ -606,7 +651,15 @@ def _install_deps(self) -> None:
             env["NO_PROXY"] = self.proxies["no_proxy"]
             env["no_proxy"] = self.proxies["no_proxy"]
 
-        # Install repo-policy-compliance package
+        old_version = execute_command(
+            [
+                "/usr/bin/pip",
```

Comment:
The check for pip in the GH actions invokes `python3 -m pip`, perhaps it can be invoked like this here as well?

Reply:
Updated the code.